File: Backend/jupddang/src/main/java/com/jupddang/jupddang/fcm/FcmTokenRequest.java
package com.jupddang.jupddang.fcm;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class FcmTokenRequest {

    private String token;
    private String deviceType;

}

File: Frontend/jupddang/lib/features/fcm/models/fcm_token_request.dart
class FcmTokenRequest {
  final String token;
  final String deviceType;

  FcmTokenRequest({
    required this.token,
    required this.deviceType,
  });

  Map<String, dynamic> toJson() {
    return {
      'token': token,
      'deviceType': deviceType,
    };
  }
}